src/Controller/LanguagesController.php

```diff
@@ -10,23 +10,23 @@
 
 class LanguagesController extends BaseApiController
 {
-    public function getLanguage(Request $request, PDO $db)
+    public function getLanguage(Request $request, PDO $db): false|array
```

Comment:
This was the result of 2 successive changes I did. Originally the function didn't have a return type, and `getLanguageById` could return false, so I set the return type to match that. Later I added an explicit check for `false`, so it will no longer be returned, but didn't update the return type to match.